accounting/bank/feeds/ponto.rst

```diff
@@ -0,0 +1,115 @@
+======================================
+Ponto as Bank Synchronization provider
+======================================
+
+**Ponto** is a service that allows companies and professionals to aggregate their accounts in one
+place and directly see all their transactions within one app.
+
+It is a third-party solution that is continuously expanding the number of bank institutions that can
+be synchronized with Odoo.
+
+.. image:: media/ponto_logo.png
+   :align: center
+   :alt: Logo of the Ponto brand
+
+**Odoo Accounting** can synchronize directly with your bank to get all bank statements imported
+automatically into your database. This allows for easier **bank reconciliation**.
+
+When :doc:`adding a bank account on Odoo <../setup/create_bank_account>`, you can see if your bank
+requires a connection through Ponto by searching for your bank institution, and clicking on it.
+
+.. image:: media/ponto_add_bank.png
+   :align: center
+   :alt: Click on a bank institution to see which third party service is required to synchronize
+         your bank with Odoo Accounting.
+
+.. note::
+   You can find more information about bank synchronization :doc:`on this page
+   <bank_synchronization>`.
+
+Configuration
+=============
+
+Link your bank accounts with Ponto
+----------------------------------
+
+#. Go on `Ponto's website (https://myponto.com) <https://myponto.com>`_.
+#. Create an account if you don’t have one yet.
+#. Once you are logged in, create an *organization*.
+
+   .. image:: media/ponto_organization.png
+      :alt: Fill out the form to add an organization in Ponto
+
+#. | Go to :menuselection:`Accounts --> Live`, and click on **Add account**.
+   | You might have to add your **Billing Information** first.
```

Comment:
```suggestion
#. Go to :menuselection:`Accounts --> Live`, and click on **Add account**. You might have to add your **Billing Information** first.
```

I did not find anything in the documentation guide that says we should do this.. so, I am not sure it should be removed, or not, I leave the comment in case you want to check or something.

Reply:
I did the whole flows myself. That's why I added this part about the Billing Information. Thanks for having checked though :)

Reply:
do you mean the line break?

That is one of my uses of the line breaks:
If I want to separate some information inside a list, then I do a line break.
Here, the info is "add account". the additional info that I want to keep apart is about the billing information.